src/main/java/core/basesyntax/models/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.models;
+
+import core.basesyntax.abstractandenum.Figure;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public int getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: " + getClass().getSimpleName() + ", area: "
+                + getArea() + " sq.units, color: " + getColor() + " ,radius: " + getRadius());
```

Comment:
```suggestion
                + getArea() + " sq.units, color: " + getColor() + " ,radius: " + radius);
```
you have direct access to the fields of this class